os/board/rtl8720e/src/component/filesystem/ftl/ftl.c

```diff
@@ -122,8 +122,6 @@ extern bool ftl_page_erase(struct Page_T *p);
 void ftl_mapping_table_init(void);
 uint16_t read_mapping_table(uint16_t logical_addr);
 
-/* Flash Status Bit */
-#define FLASH_STATUS_BITS 0x2c
```

Comment:
To reduce the changes maybe you can just change the value to CONFIG_FLASH_STATUS_BITS .